### ISSUE TITLE
Do you have a comparison to CDVAE?

### ISSUE THREAD

**Issue by sgbaird** [MilesZhao/PGCGM#2]
I noticed you compared results to FTCP. Do you have comparison results for [CDVAE](https://github.com/txie-93/cdvae)? I understand this might be less feasible for the 500k generation based on the longer time it takes to generate CDVAE samples.

**Comment by usccolumbia** [comment]
" I understand this might be less feasible for the 500k generation based on the longer time it takes to generate CDVAE samples".  Yes you said it. 

Yes. We just finished the comparison on a smaller scale. The major finding is that CDVAE is not good at generating high-symmetry crystal structures. We generated 1100 structures using CDVAE. 860 are pymatgen readable structures. Out of the top 10 space groups with most samples, 97.8% 773/790  sampleshave very low space group ID numbers (<=25). This is a serious issue as most stable structures tend to have high symmetry.
